packages/cli/package.json

```diff
@@ -19,6 +19,7 @@
     "cli-ux": "^5.5.1",
     "coveo.analytics": "^2.18.4",
     "create-react-app": "^4.0.3",
+    "dedent": "^0.7.0",
```

Comment:
Neat library to avoid oopsies in indentation with template string
https://www.npmjs.com/package/dedent